packages/customers/src/Nav.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FC, useEffect } from 'react';
+import { useMatch } from 'react-router-dom';
+import {
+  Customers,
+  Collapse,
+  List,
+  useDrawer,
+  AppNavLink,
+} from '@openmsupply-client/common';
+
+const useNestedNav = (path: string) => {
+  const { open } = useDrawer();
+  const match = useMatch(path);
+  const [expanded, setExpanded] = React.useState(false);
+
+  useEffect(() => {
+    setExpanded(!!match);
+  }, [match]);
+
+  return { isActive: open && expanded };
+};
+
+const Nav: FC = () => {
+  const { isActive } = useNestedNav('customer/*');
+
+  return (
+    <>
+      <AppNavLink to="customers" icon={<Customers />} text="Customers" />
+      <Collapse in={isActive}>
+        <List>
+          <AppNavLink
+            to="/customers/customer-invoice"
```

Comment:
`customer-invoices`?

Reply:
or just `invoices`? it is under customers after all.

Reply:
is it time to add a global enum for routes? I see `customer-invoice` a number of times already in this PR

Reply:
yes - issue so I can merge this https://github.com/openmsupply/openmsupply-client/issues/43